data-types.md

```diff
@@ -104,7 +104,7 @@ Events represent changes in vehicle status.
 
 (?) Do we still want `event_geographies`?
```

Comment:
Yes it's come up again with some new modes. https://github.com/openmobilityfoundation/mobility-data-specification/discussions/808